addon/mixins/run.js

```diff
@@ -53,6 +59,10 @@ export default Mixin.create({
     this._pendingDebounces = undefined;
     this._pendingThrottles = undefined;
     this._pollerTokens = undefined;
+
+    this._isRunTaskDisposableRegistered = false;
```

Comment:
Hmm, I don't think we need this specific flag on the instance. If we move the weakmap from `getPendingTimers` into this file, we can just check it below with a `.has` check.

Reply:
👍